ai_presenter/database.py

```diff
@@ -20,6 +23,10 @@ def get_config(self) -> Config:
     def get_data(self):
         return self.alldata
 
+    def check_valid(self):
```

Comment:
behavior of how we handle scene limit is something we should specify, do we still want the program to run with 10 scenes and cut off the rest or do we make the scene generation restart? or could we specify in chatgpt